### ISSUE TITLE
Add info on Out-File output width to the about_Redirection and Out-File help topics

### ISSUE THREAD

**Issue by rkeithhill** [MicrosoftDocs/PowerShell-Docs#7651]
# Documentation Issue

PowerShell users have run into issues for years when logging output to a file using either `Out-File` or `>` where the data gets clipped based on the current console's width (or 80 as a fallback if PS can't determine with width).  This is unfortunate because lines in files have no such width limitation and more importantly, useful data gets clipped in the file e.g.:

```powershell
Get-ChildItem Env:\Path | Out-File path.log
Get-Content .\path.log

Name                           Value
----                           -----
Path                           C:\Program Files\PowerShell\7-preview;C:\WINDOWS…
```

The gist of the addition help text is to indicate the user can set this at the top of their script to get wider formatted lines of output in files:

```powershell
$PSDefaultParameterValues['out-file:width'] = 10000
```

## Context of the issue

- URL to the article (Docs or GitHub URL): https://github.com/PowerShell/PowerShell/issues/15404
- Issue affects the following content (check all that apply):

**Conceptual content**
- [ ] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [ ] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Sample scripts
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Legacy SDK

**Cmdlet reference & about_ topics**
- [x] Preview content
- [x] Version 7.1 content
- [x] Version 7.0 content
- [x] Version 5.1 content

- Is the issue specific to a platform (Y/N - Win/macOS/Linux): No

## Detailed description of the issue

I plan on submitting PR to add this info but need to know if I should target just preview or 5.1, 7.0 and 7.1 as well?

Initiated from discussion on https://github.com/PowerShell/PowerShell/issues/15404

**Comment by chasewilson** [comment]
Hey @rkeithhill, thanks for this! 

You can target all four versions of the Out-File and about_redirection docs if you don't mind.
The suggestion would be to add information to the **Width** parameter.

I'll be keeping an eye out for your PR to get it merged in quickly.

**Comment by sdwheeler** [comment]
Closed by #7656
